assets/js/template-dom-scripts.js

```diff
@@ -104,23 +104,31 @@
   }
 
   function applyDarkTheme() {
-    var rules = [
-      '.intro-and-nav, .main-and-footer { filter: invert(100%); }',
-      '* { background-color: inherit; }',
-      'img:not([src*=".svg"]), .colors, iframe, .demo-container { filter: invert(100%); }'
-    ];
-    rules.forEach(function(rule) {
-      document.styleSheets[0].insertRule(rule);
-    })
+    var darkTheme = document.getElementById('darkTheme');
+    darkTheme.disabled = false;
   }
 
   function clearDarkTheme() {
-    for (let i = 0; i < document.styleSheets[0].cssRules.length; i++) {
-      document.styleSheets[0].deleteRule(i);
+    var darkTheme = document.getElementById('darkTheme');
+    darkTheme.disabled = true;
+  }
+
+  function defaultDarkTheme() {
+{{- if and (isset .Site.Params "defaultdarktheme") (.Site.Params.defaultDarkTheme) }}
```

Comment:
```suggestion
{{- with .Site.Params.defaultDarkTheme }}
```

Reply:
Thanks. I thought for some reason that the with would through and error if the param `defaultDarkTheme` didn't exist, but it doesn't, which is good to know for future.